main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -1457,6 +1457,8 @@ object Parser2 {
              | TokenKind.LiteralRegex => literalExpr()
         case TokenKind.ParenL => parenOrTupleOrLambdaExpr()
         case TokenKind.Underscore => if (nth(1) == TokenKind.ArrowThinR) unaryLambdaExpr() else name(NAME_VARIABLE, context = SyntacticContext.Expr.OtherExpr)
+        case TokenKind.NameLowerCase | TokenKind.NameUpperCase if nth(1) == TokenKind.Hash =>
```

Comment:
What's the reason for allowing uppercase name?

Reply:
I don't quite understand. In Flix, all variables are lowercase.

Reply:
I see, I didn't know.
What are the ways to create/manipulate java objects in Flix?
In this case, only the java object should be lower case, if you agree.
The java methods could be case-free.

Reply:
I think we should simply only have the lowercase.